truss-chains/truss_chains/framework.py

```diff
@@ -47,6 +47,12 @@
 _DOCS_URL_LOCAL = "https://docs.baseten.co/chains/guide#local-development"
 _DOCS_URL_STREAMING = "https://docs.baseten.co/chains/guide#streaming"
 
+# A "neutral dummy" endpoint descriptor if validation fails, this allows to safely
```

Comment:
This was duplicated twice below when validating the endpoint, small refactor to consolidate up above